boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-memory_map.dtsi

```diff
@@ -8,14 +8,14 @@
 	reserved-memory {
 		cpuapp_ram0x_region: memory@2f000000 {
 			compatible = "nordic,owned-memory";
-			reg = <0x2f000000 DT_SIZE_K(260)>;
+			reg = <0x2f000000 DT_SIZE_K(768)>;
```

Comment:
Just grabbing all that for the application domain is a bit heavy handed, given this now covers also the ipc regions for app<->radio and radio<->secure. 
This will lead to overlaps in the memory allocations between application domain and radio and result in a conflict during boot

Reply:
yes, i did it only to make it compatible with SDFW i was testing with. It will need to be aligned later.

Reply:
resolved by rebasing on  latest zephyr